.github/workflows/continuous-integration.yml

```diff
@@ -156,9 +153,8 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-          # TODO: Enable with pull_request_target to allow for forked repos to run our github actions
-          # ref: ${{github.event.pull_request.head.ref}}
-          # repository: ${{github.event.pull_request.head.repo.full_name}}
+          ref: ${{github.event.pull_request.head.ref}}
```

Comment:
This says `pull_request` and we have updated `pull_request` to `pull_request_target` under `on:`, just checking this is correct right?

Reply:
Yeah, the event type is still pull_request